web/packages/teleterm/src/ui/DocumentsReopen/DocumentsReopen.story.tsx

```diff
@@ -26,55 +24,47 @@ export default {
 
 export const Story = () => {
   return (
-    <MockAppContextProvider>
```

Comment:
Wow, nice, I'd definitely not remember to remove the context provider from here.